src/OpenTelemetry.Exporter.Zipkin/Implementation/ZipkinSpan.cs

```diff
@@ -317,5 +317,18 @@ public void Write(Utf8JsonWriter writer)
         }
 
 #endif
+
+        private string ConvertObjectToString(object obj)
```

Comment:
Nice refactor! Let's add `AggressiveInlining` hint to this?

Reply:
done!